arelle/PluginManager.py

```diff
@@ -84,6 +84,7 @@ def save(cntlr):
         pluginConfigChanged = False
     
 def close():  # close all loaded methods
+    pluginConfig.clear()
```

Comment:
I haven't looked too hard, but if pluginConfig can be `None` (line 45), can this line throw an exception?

Reply:
Disregard, looks to not be an issue.